python/excludeVMs/excludeVMs.py

```diff
@@ -36,7 +36,7 @@
     excludeRules = []
 
 if excludePoweredOff is True:
-    if vcuser is None or vpassword is None:
```

Comment:
I was getting the following warning:
`SyntaxWarning: "is" with a literal. Did you mean "=="?`
changed `is` to `==` and it cleared up.

Reply:
strangely, I reverted back to `is` and it still worked. Not sure why. 
closing PR because it worked with `is`